### PR TITLE
[test] Infer before export

### PR DESCRIPTION
**Pull request by dayo09**
Let's infer before export, to avoid the model's internal state change due to 'export leakage'.

TICO-DCO-1.0-Signed-off-by: Dayoung Lee <dayoung.lee@samsung.com>

---

#129

### CODE REVIEW

File: test/pt2_to_circle_test/test_pt2_to_circle.py [hunk 1]
```diff
@@ -140,6 +140,10 @@ def verify_circle(circle_model_path: str, opt_circle_model_str: str):
 @print_name_on_exception
 def infer_nnmodule(model: torch.nn.Module, example_inputs: tuple):
     with torch.no_grad():
+        # Model should be frozen to compare the result with others.
+        # e.g. BatchNorm running_mean/running_var will be updated during training mode, thus changing the model behavior.
+        model.eval()
```
Comment (dayo09) [inline test/pt2_to_circle_test/test_pt2_to_circle.py:145]: FOR OTHER REVIEWERS

[Seunghui98](https://github.com/Seunghui98) [1 hour ago](https://github.com/Samsung/TICO/pull/120#discussion_r2128140617)
it's just out of curiousity. 😀
If this line is added, does it not affect other tests?


@[dayo09](https://github.com/dayo09) dayo09 [34 minutes ago](https://github.com/Samsung/TICO/pull/120#discussion_r2128214059)
@Seunghui98 That's a good question.

It seems only BatchNorm family within our test cases are affected - only they fails the test.
WHY? - BatchNorm changes its internal values during inference if it's not set as eval() mode (training mode)
WHY It didn't failed before then? - Before, eval() was set outside this function (in convert_nnmodule_to_pt2), but this PR includes the change that convert_nnmodule_to_pt2 is run on copied model.

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by seockho-kim — APPROVED** [review]
LGTM
